runner.py

```diff
@@ -1120,7 +1124,17 @@ def stop_metric_providers(self):
 
     def read_and_cleanup_processes(self):
         print(TerminalColors.HEADER, '\nReading process stdout/stderr (if selected) and cleaning them up', TerminalColors.ENDC)
-        process_helpers.kill_ps(self.__ps_to_kill)
+
+        ps_errors = []
+        for ps in self.__ps_to_kill:
+            try:
+                process_helpers.kill_ps(ps['ps'], ps['ps'].pid, ps['cmd'])
+            except ProcessLookupError as exc:
```

Comment:
Can't this also be a RunTimeError?

Reply:
The idea here is to let the RuntimeError bubble up and fail the path. Having a process not available anymore is very often expected and metrics are fine. But having to SigKill is an error and metrics are most likely corrupted.

To let the user see this I choose for a harder error. Clairfied this with a comment